CleanWeatherTests/Mocks/Mock+CityForecast.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Mock+CityForecast.swift
+//  CleanWeatherTests
+//
+//  Created by Marek Skrzelowski on 27/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+@testable import CleanWeather
+
+extension Mock {
+
+    static let cityForecast1 = CityForecast(id: "c1f2934e-b2f4-47d2-961f-22c72d2411b5", hour: "12", hourTemp: 10.34, hourPrecipProbability: 0.46, icon: "clear-day")
```

Comment:
do we need to have hour as a string? that limits some features that we could use, for example sorting